### PR TITLE
feature(newman): add newman reporter

### PR DESCRIPTION
**Pull request by vovsemenv**
What have we done:
1. Move old package to monorepo from old https://github.com/ervirendersingh/newman-reporter-allure .
2. Cleanup files
3. Fix some issues
4. Rewrite to typescript
5. Add metadata features
6. Add tests

### CODE REVIEW

File: packages/allure-newman/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "newman-reporter-allure",
```
Comment (baev) [inline packages/allure-newman/package.json:2]: allure-newman

Comment (vovsemenv) [reply packages/allure-newman/package.json:2]: there is a naming convention for newman reporters https://github.com/postmanlabs/newman#creating-your-own-reporter

File: packages/newman-reporter-allure/src/helpers.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { Label, LabelName } from "allure-js-commons";
+import { EventList } from "postman-collection";
+
+export const extractMeta = (eventList: EventList) => {
+  const labels: Label[] = [];
+
+  const allureIdRegexp = /\/\/\s*@allure:id=(.+)/;
```
Comment (baev) [inline packages/newman-reporter-allure/src/helpers.ts:7]: We have the following format:

```java
ID_TAG = Pattern.compile("^@?allure\\.id[:=](?<id>.+)$");
LABEL_TAG = Pattern.compile("^@?allure\\.label\\.(?<name>.+)[:=](?<value>.+)$");
```

**Review by baev — CHANGES_REQUESTED** [review]

File: packages/newman-reporter-allure/src/helpers.ts (hunk 2 above)
Comment (vovsemenv) [reply packages/newman-reporter-allure/src/helpers.ts]: thx, fixed

**Review by baev — APPROVED** [review]

**Pull request merged**